### ISSUE TITLE
!closeas @user

### ISSUE THREAD

**Issue by NexExMachina** [Dragory/modmailbot#433]
Would be a nice feature to credit closes to specific users who handled tickets.

**Comment by Dragory** [comment]
So to verify, you'd like to be able to credit closes in the log channel ("Thread with X was closed by <user>") to a specific user other than yourself?

**Comment by NexExMachina** [comment]
correct. This will allow us to give the appropriate staff member their ticket close

**Comment by NexExMachina** [comment]
How will I know if this gets accepted or rejected?
